CMakeLists.txt

```diff
@@ -347,6 +347,7 @@ if(${CMAKE_CXX_COMPILER_ID} MATCHES "GNU" OR
         endif()
     else() # not APPLE
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
```

Comment:
In GCC 6.1, the default c++ standard is c++14; maybe that's why? I think this should still have the `-std=c++11`. If you want to enable all warnings, set `CMAKE_CXX_FLAGS` in the CMake GUI / ccmake / command line (`cmake . -DCMAKE_CXX_FLAGS="-Wall"`).